dynamodb/src/main/java/site/ycsb/db/DynamoDBClient.java

```diff
@@ -286,6 +298,11 @@ public Status insert(String table, String key, Map<String, ByteIterator> values)
       attributes.put(hashKeyName, new AttributeValue(hashKeyValue));
     }
 
+    if (null != this.ttlKeyName) {
+      attributes.put(this.ttlKeyName, new AttributeValue(
+          String.valueOf((System.currentTimeMillis() / 1000L) + this.ttlDuration)));
```

Comment:
One suggestion by Nadav was testing multiple TTL-duration values.
It may not be a must P1 but IIUC it can't fit to this implementation.

Reply:
Not sure what multiple mean in this context.

if you mean multiple row with exactly the same TTL value ?, I think that can be done by updating the row you want, and not necessary something to do on while inserting the rows.

Reply:
If you mean that we would want different values for different rows, we will probably start a new ycsb stress for different range of the rows and set their a different value.

Reply:
@yarongilor probably refers to the idea I mentioned of having both items that **should** expire (e.g., in 30 minutes) and items that **shouldn't** expire during the test - just to test that we don't accidentally delete items which we shouldn't.
@roydahan makes a very good point that we don't need special support for that - we can just run two separate YCSB runs, one which adds a bunch of non-expiring data, and another one which adds the bulk of the expiring data.